### PR TITLE
universal_address: minor dox fixes

### PR DESCRIPTION
**Pull request by cgundogan**

### CODE REVIEW

File: sys/include/universal_address.h [hunk 1]
```diff
@@ -28,30 +28,30 @@ extern "C" {
...
     uint8_t use_count;                       /**< The number of entries link here */
-    uint8_t address_size;                    /**< Size in bytes of the used genereic address */
-    uint8_t address[UNIVERSAL_ADDRESS_SIZE]; /**< the genereic address data */
+    uint8_t address_size;                    /**< Size in bytes of the used generic address */
+    uint8_t address[UNIVERSAL_ADDRESS_SIZE]; /**< The generic address data */
 } universal_address_container_t;
 
 /**
- * @brief initializes the datastructure for the entries
+ * @brief Initializes the data structure for the entries
```
Comment (OlegHahm) [inline sys/include/universal_address.h]: While you're at it: didn't we conclude that the third person 's' should be omitted?

Comment (cgundogan) [inline sys/include/universal_address.h]: I remember the discussion .. but I cannot remember a conclusion?

Comment (cgundogan) [inline sys/include/universal_address.h]: However, I can address

**Comment by BytesGalore** [comment]
@cgundogan thx for the fixes, ACK

**Comment by cgundogan** [comment]
removed third person 's' and squashed immed.

**Pull request merged**